ui/decredmaterial/editor.go

```diff
@@ -51,6 +60,20 @@ func (t *Theme) EditorPassword(editor *widget.Editor, hint string) Editor {
 	return e
 }
 
+func (t *Theme) RestoreEditor(editor *widget.Editor, hint string, title string) RestoreEditor {
+
+	e := t.Editor(editor, hint)
+	e.Bordered = false
+	return RestoreEditor{
+		t:          t,
+		Edit:       e,
+		TitleLabel: t.Body2(title),
+		LineColor:  t.Color.Gray1,
+		m10:        values.MarginPadding10,
```

Comment:
You can use the dimension directly in the code as a unit.Dp value. One of the rational behind the decredmaterial package was to have it independent of godcr code such that it can easily be pulled out and used by any other project.